workflows/chipseq/chipseq-standard.wdl

```diff
@@ -1,32 +1,5 @@
-## # ChIP-Seq Standard
-##
-## This WDL workflow runs the BWA ChIP-seq alignment workflow for St. Jude Cloud.
-##
-## The workflow takes an input BAM file and splits it into FASTQ files for each read in the pair.
-## The read pairs are then passed through BWA alignment to generate a BAM file.
-## File validation is performed at several steps, including immediately preceeding output.
-##
-## ## LICENSING
-##
-## #### MIT License
-##
-## Copyright 2021-Present St. Jude Children's Research Hospital
-##
-## Permission is hereby granted, free of charge, to any person obtaining a copy of this
-## software and associated documentation files (the "Software"), to deal in the Software
-## without restriction, including without limitation the rights to use, copy, modify, merge,
-## publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons
-## to whom the Software is furnished to do so, subject to the following conditions:
-##
-## The above copyright notice and this permission notice shall be included in all copies or
-## substantial portions of the Software.
-##
-## THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
-## BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
-## NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
-## DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-## OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-
+# SPDX-License-Identifier: MIT
+# Copyright St. Jude Children's Research Hospital
```

Comment:
I'm not sure about the shortened header for workflows. I'm not sure our earlier headers were good, but I think a brief overview of the workflow and purpose is a reasonable inclusion.

Reply:
I'm open to the possibility. My counter-argument would be that the "brief overview of the workflow and purpose" would better belong in `meta.description`. I kept those brief, probably too brief. I'd rather rewrite those to be more verbose than go back to the headers.

What do you think of that?

Reply:
I'm fine with that as well. I think many people are conditioned to look at the start of a file for some information, but since WDL provides a mechanism for documentation elsewhere, I don't think we need to repeat it.